clients/admin-ui/cypress/e2e/systems.cy.ts

```diff
@@ -414,6 +434,11 @@ describe("System management page", () => {
     beforeEach(() => {
       stubSystemCrud();
       stubTaxonomyEntities();
+      cy.fixture("systems/systems.json").then((systems) => {
+        cy.intercept("GET", "/api/v1/system/*", {
+          body: systems[0],
+        }).as("getFidesctlSystem");
+      });
```

Comment:
note that we had different fixtures for 1️⃣  getting a list of systems vs 2️⃣  getting just one system. the tests were pretty dependent on the first system from the list of systems (1️⃣ ), so I made sure to keep that around and index into it, rather than relying on 2️⃣ which had data the tests didn't expect and would've required more changes